applications/nrf_desktop/src/hw_interface/leds.c

```diff
@@ -128,17 +128,17 @@ static void led_update(struct led *led)
 static int leds_init(void)
 {
 	const char *dev_name[] = {
-#if CONFIG_PWM_0
-		DT_NORDIC_NRF_PWM_PWM_0_LABEL,
+#if defined(CONFIG_PWM) && DT_NODE_HAS_STATUS(DT_NODELABEL(pwm0), okay)
+		DT_LABEL(DT_ALIAS(pwm_0)),
```

Comment:
I sometimes wonder... why? Who thought that instead of MACRO_NAME its better to go with A(B(NAME)) and even better to put this easy to use and totally not bug-prone thing that you now have one line above.

Reply:
The new API enables various features that the old direct-access-to-generated-macros style made impossible. Sorry you find it unreadable. Hopefully once you familiarize yourself with it, you'll start to see the benefits. Please let me know if you have any big concerns. Thanks.